web/src/components/AuthResults/AuthResults.js

```diff
@@ -17,13 +17,16 @@ export default () => {
         currentUser:
         <pre style={{ margin: 0 }}>{JSON.stringify(currentUser, null, 2)}</pre>
       </code>
-      <h3>Basic sanity checks</h3>
```

Comment:
I'm interested in hearing about why you made the changes you made to this file

Reply:
I updated it to hide the "Basic sanity checks" header and removed "Login first" to declutter the page a bit more.

Reply:
Also made an update where it instead checks if the user is authenticated before showing this section in case they are authenticated and the `userMetadata` is still somehow null.

Reply:
👍